### PR TITLE
[#6 Software Updates] Adding Store

### PR DESCRIPTION
**Pull request by MatApple**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

File: src/js/stores/ServicePlanStore.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+      servicePlanListeners[serviceID].forEach(callback =>
+        callback(this.getServicePlan(serviceID))
+      );
+    }
+    // Emit event for StoreConfig Listeners
+    this.emit(PLAN_CHANGE, serviceID);
+  },
+
+  processPlanFetchError: function (error, serviceID) {
+    ServicePlanStore.emit(PLAN_ERROR, error, serviceID);
```
Comment (jfurrow) [inline src/js/stores/ServicePlanStore.js]: Can we just use `this.emit` here?

**Comment by jfurrow** [comment]
Can we add this store to `constants/StoreConfig.js`?

File: tests/_fixtures/plan/plan.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          "name": "broker-2",
+          "status": "Pending"
+        }
+      ],
+      "id": "59a843b3-e4e7-4a2d-b77b-803bb704a52a",
+      "name": "Update to: 0ca3aed4-02f1-4175-8399-61e98676a8d7",
+      "status": "InProgress"
+    }
+  ],
+  "status": "InProgress"
```
Comment (jfurrow) [inline tests/_fixtures/plan/plan.json]: Do we need to add a `plan` key to this fixture?

File: src/js/stores/ServicePlanStore.js (hunk 1 above)
Comment (MatApple) [inline src/js/stores/ServicePlanStore.js]: Absolutely not!! Haha good catch :) Fixed

**Comment by MatApple** [comment]
@jfurrow Added to StoreConfig 👍

**Comment by jfurrow** [comment]
LGTM!

**Comment by kennyt** [comment]
can you rebase this?

**Comment by kennyt** [comment]
i believe this needs another rebase :)

**Pull request merged**